.github/CODEOWNERS

```
@@ -1 +1,6 @@
 * ryohsuke.mitsudome@tier4.jp mfc@leodrive.ai
+
+.devcontainer/** oguz@leodrive.ai yutaka.kondo@tier4.jp
+.github/** oguz@leodrive.ai yutaka.kondo@tier4.jp
+ansible/** oguz@leodrive.ai yutaka.kondo@tier4.jp
+docker/** oguz@leodrive.ai yutaka.kondo@tier4.jp
```

Is @oguzkaganozt's email address for GitHub wrong?
```suggestion
.devcontainer/** oguzkaganozt@gmail.com yutaka.kondo@tier4.jp
.github/** oguzkaganozt@gmail.com yutaka.kondo@tier4.jp
ansible/** oguzkaganozt@gmail.com yutaka.kondo@tier4.jp
docker/** oguzkaganozt@gmail.com yutaka.kondo@tier4.jp
```
https://github.com/oguzkaganozt

---

I've contacted him, he will add the `@leodrive.ai` email to his github account.

---

I got it.

---

Seems he will transition to AWF, so will keep the gmail address for now.